apps/chat/src/components/Common/ShareIcon.tsx

```diff
@@ -65,21 +69,22 @@ export default function ShareIcon({
         className={classNames(
           'absolute -bottom-1 -left-1 bg-layer-3',
           isPublished ? 'rounded-md' : 'rounded-sm',
+          iconWrapperClassName,
         )}
         data-qa={
           isPublished && isPublishingEnabled ? 'world-icon' : 'arrow-icon'
         }
       >
-        <Tooltip tooltip={t(isPublished ? 'Published' : 'Shared')}>
+        <Tooltip tooltip={t(isPublished ? t('Published') : t('Shared'))}>
```

Comment:
it's already wrapped in translation
```suggestion
        <Tooltip tooltip={t(isPublished ? 'Published' : 'Shared')}>
```